xbmc/interfaces/json-rpc/AudioLibrary.cpp

```diff
@@ -1173,6 +1189,7 @@ JSONRPC_STATUS CAudioLibrary::GetAdditionalSongDetails(const CVariant &parameter
   checkProperties.insert("albumartist");
   checkProperties.insert("albumartistid");
   checkProperties.insert("musicbrainzalbumartistid");
+  checkProperties.insert("disctitle");
```

Comment:
Disc subtitle is not an additional property of a song from some other table, it is fetched like mood or (song) title. This approach does not seem right